histomicstk/features/ComputeFSDs.py

```diff
@@ -0,0 +1,179 @@
+import collections
+import numpy as np
+from skimage.segmentation import find_boundaries
+
+
+def ComputeFSDs(Label, K=128, Fs=6):
+    """
+    Calculates `Fourier shape descriptors` from a label mask.
+
+    Parameters
+    ----------
+    Label : array_like
+        A M x N label mask.
```

Comment:
I think if we modify the function to accept a labeled mask with multiple object and compute features for all the objects then we will end up calling find_boundaries only once for the whole labeled mask